examples/plot_glm_retinal_ganglia.py

```diff
@@ -0,0 +1,264 @@
+# In this case, we will pick cell number 2 (ON cell).
+
+cell_idx = 2  # pick cell number 2 (ON cell)
+spike_time = rgcs_dataset['spike_times']['cell_%d' % cell_idx]
+spike_time = np.array(spike_time)
+n_spikes = len(spike_time)  # number of spikes
+
+# bin the spikes
+t_bins = np.arange(n_times + 1) * dt
+spikes_binned, _ = np.histogram(spike_time, t_bins)
+
+print('Loaded RGC data: cell {}'.format(cell_idx))
+print('Number of stim frames: {:d} ({:.1f} minutes)'.
+      format(n_times, n_times * dt / 60))
+print('Time bin size: {:.1f} ms'.format(dt * 1000))
+print('Number of spikes: {} (mean rate = {:.1f} Hz)\n'.
+      format(n_spikes, n_spikes / n_times * 60))
+
+########################################################
+#
+# Visualize first 120 samples of the stimulation
+# and the spiking pattern.
+sample_idx = np.arange(120)
+t_sample = dt * sample_idx
+tmax = t_sample.max()
+
+fig, axes = plt.subplots(3, 1, sharex=True)
+axes[0].step(t_sample, stim[sample_idx])
+axes[0].set_xlim([0., tmax])
+axes[0].set_title('Raw Stimulus, spikes, and'
+                  ' spike counts')
+axes[0].set_ylabel('Stimulation Intensity')
+
+tspplot = spike_time[(spike_time >= t_sample.min()) &
+                     (spike_time < t_sample.max())]
+markerline, _, _ = axes[1].stem(spike_time[sample_idx],
+                                [1] * len(spike_time[sample_idx]))
+markerline.set_markerfacecolor('none')
+axes[1].set_xlim([0., tmax])
+axes[1].set_ylim([0, 2])
+axes[1].set_ylabel('Spikes')
+
+markerline, _, _ = axes[2].stem(t_sample, spikes_binned[sample_idx])
+markerline.set_markerfacecolor('none')
+axes[2].set_xlim([0., tmax])
+axes[2].set_xlabel('Time (s)')
+axes[2].set_ylabel('Binned Spike counts')
+plt.show()
+
+########################################################
+#
+# We can use ``scipy``'s function ``hankel`` to make our design matrix.
+# Design matrix :math:`X` can be created using the stimulation and its history.
+# Later in the tutorial, we will also incorporate spikes history into our
+# design matrix.
+
+n_t_filt = 25  # tweak this to see different results
+stim_padded = np.pad(stim, (n_t_filt - 1, 0))
+Xdsgn = hankel(stim_padded[0: -n_t_filt + 1], stim[-n_t_filt:])
+
+plt.figure()
+plt.imshow(Xdsgn[:50, :],
+           cmap='binary',
+           aspect='auto',
+           interpolation='nearest')
+plt.xlabel('lags before spike time')
+plt.ylabel('time bin of response')
+plt.title('Sample first 50 rows of design'
+          ' matrix created using Hankel')
+plt.show()
+
+########################################################
+# **Fitting and predicting with a linear-Gaussian GLM**
+#
+# For a general linear model, an observed spikes can be
+# thought of an underlying parameter
+# :math:`\beta_0, \beta` that control the spiking:
+# :math:`y = (\beta_0 + X \beta) \ + \epsilon` and
+# :math:`y \sim \text{Poiss}(\beta_0 + X \beta)` where
+# :math:`X` is the stimulation and history of stimulation
```

Comment:
I wonder if we should get rid of this math here. People can refer to the tutorial if they're interested in more details.

Reply:
I kinda want to keep it since it can be read through without moving out of the example.

Reply:
Equations removed